src/cli/require-templates.js

```diff
@@ -29,14 +29,18 @@ export const requireTemplates = function requireTemplates(store) {
   // Initialize templates, js, css and HMR acceptance logic
   const prepend = `
 var store = require('./huron-store.js');
+var sectionTemplate = require('./section.hbs');
 var assets = require.context(${requirePath}, true, ${requireRegex});
 var modules = {};
 
+modules['${store.get('sectionTemplatePath')}'] = sectionTemplate;
+
 assets.keys().forEach(function(key) {
   modules[key] = assets(key);
 });
 
 if (module.hot) {
+  // HMR for huron components (json, hbs, html)
```

Comment:
For clarity, I wonder if it makes sense replace instances of 'HMR' with 'Hot Module Replacement'.